Sources/web3swift/Web3/Web3+HttpProvider.swift

```diff
@@ -3,6 +3,9 @@
 //  Copyright © 2018 Alex Vlasov. All rights reserved.
 //
 
+#if canImport(FoundationNetworking)
+import FoundationNetworking
+#endif
```

Comment:
And this one should be refactored to not use URLSession in it's i it (it was a rough one implementation within 3.0.0 release, so it's could be changed).